pyproject.toml

```diff
@@ -6,10 +6,10 @@ authors = ["Martin Keogh <martin1keogh@gmail.com>"]
 
 [tool.poetry.dependencies]
 python = ">=3.8, <3.10"
-pybigquery = "^0.6.0"
+pybigquery = "^0.5.0"
```

Comment:
Obviously ~incorrect.  
Actually requires applying https://github.com/googleapis/python-bigquery-sqlalchemy/pull/84 locally for the tests to pass.